lib/create-plugin.js

```diff
@@ -427,4 +420,63 @@ function isIntrospectionQuery(operation, captureIntrospectionQueries) {
   })
 }
 
+function isServiceDefinitionQuery(operation, captureServiceDefinitionQuery) {
```

Comment:
the docs in README state the new config options default to false but I don't see that happening around here https://github.com/newrelic/newrelic-node-apollo-server-plugin/pull/112/files#diff-cdfa5918886b4e4d7e6b99e9685bb5118e2e2479a2e6f10993c00356677ff6dfR65

Reply:
Defaulted the config items and updated the `ignore` logic. Let me know what you think.